frontend/src/components/message/MessageDisplay.tsx

```diff
@@ -20,12 +20,16 @@ export const MessageDisplay = ({
   // 新規メッセージを追加
   useEffect(() => {
     setMessages((prev) => {
-      const newMessages = defaultMessages.filter((message) => {
-        const isExist = prev.find(
-          (prevMessage) => prevMessage.id === message.id,
-        );
-        return !isExist;
-      });
+      const newMessages = defaultMessages
+        .filter((message) => {
+          const isExist = prev.find(
+            (prevMessage) => prevMessage.id === message.id,
+          );
+          return !isExist;
+        })
+        .filter((message) => {
+          return message.createdAt > Date.now() / 1000 - 10;
```

Comment:
こんな感じで引数として受け取ると、マジックナンバー（意味が謎な値）にならなくて良さそうです！
```suggestion
          return message.createdAt > Date.now() / 1000 - bufferDurationInSec;
```

```
export const MessageDisplay = ({
  messages: defaultMessages,
  bufferDurationInSec = 10
}: {
  messages: Message[];
  bufferDurationInSec: number
}) => {
```